src/Countries/US/Georgia/GeorgiaIncome/V20180101/GeorgiaIncome.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\Georgia\GeorgiaIncome\V20180101;
+
+use Appleton\Taxes\Classes\Payroll;
+use Appleton\Taxes\Countries\US\Georgia\GeorgiaIncome\GeorgiaIncome as BaseGeorgiaIncome;
+use Appleton\Taxes\Countries\US\FederalIncome\FederalIncome;
+use Appleton\Taxes\Models\Countries\US\Georgia\GeorgiaIncomeTaxInformation;
+
+class GeorgiaIncome extends BaseGeorgiaIncome
+{
+    const SUPPLEMENTAL_TAX_RATE = 0.05;
```

Comment:
This looks like the one from AL. Should we remove the bonus tax rate stuff or maybe update it for each state? I'm just not sure we should leave it in place, in case it accidentally gets used without being updated.

![sup](https://user-images.githubusercontent.com/1130888/37989711-dbb0ef94-31c9-11e8-83d2-195d2dbe870e.png)

Source: https://dor.georgia.gov/sites/dor.georgia.gov/files/related_files/document/TSD/Guide/2018_Employers_Tax_Guide.pdf